_posts/2021-07-19-changelog-151.markdown

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #151"
+date: 2021-07-19 12:00:00 +0300
+---
+
+
+## New Features
+
+* [#6392] Add `Unused import` inspection. Note, this inspection is currently disabled by default due to possible false-positive warnings.
+  You can enable it via `Preferences | Editor | Inspections` [settings](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-code-analysis.html#inspection-settings) (by [@Kobzol])
+    <!-- Maybe get rid of these two items? -->
+    * [#7507] Support declarative macros
+
+    * [#7461] Correctly handle `writeln!` macro
+
+* [#7432] Confirm loading untrusted Cargo project
+  <!-- TODO: more information? -->
+  <!-- TODO: gif or screenshot? -->
+  
+* [#6342] Add completion for trait methods, types, and constants inside `impl` blocks (by [@Kobzol])
+
+  {% include gif-img.html path="/assets/posts/changelog-151/trait-methods-completion" w="700px" %}
+  
+* [#6664] Substitute generic parameters in [`Parameter Info`](https://www.jetbrains.com/help/idea/viewing-reference-information.html#view-parameter-info) (by [@Kobzol])
+
+    <img src="/assets/posts/changelog-151/parameter-info.png" alt="Parameter info" width="700px">
+
+* `.match` [postfix template](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-code-generation.html#postfix-completion) improvements:
+
+  * [#7476] Always place caret into the first match arm
+
+  * [#7346] Insert `.as_str()` automatically when matching `String` (by [@Kobzol])
+
+  * [#7328] Allow filling pats after generating match arms (by [@Kobzol])
+
+* [#7425] Enable crates local index (`org.rust.crates.local.index` [experimental feature]) on the nightly plugin channel by default. The new index is aimed to improve completion and code insight related to crate dependencies, mainly affecting `Cargo.toml` files.
+  If you face any issues, please feel free to provide us with feedback via `Help | Create New Issue` action right from your IDE
+
+* [#4109] Provide LLDB pretty-printers to render `OsStr`, `CString`, `CStr`, `PathBuf`, `Path` std types in IDE debugger
+
+    <img src="/assets/posts/changelog-151/lldb-pretty-printers.png" alt="LLDB pretty-printers" width="700px">
+
+* [#7148] Add `New crate version available` inspection for `Cargo.toml` files. This inspection warns about possibly outdated dependency versions.
```

Comment:
Maybe add a common top level item about crate local index since we have several items about it